compio-log/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+#[doc(hidden)]
+pub use tracing::*;
```

Comment:
Seems that `instrument` macro is enabled no matter `enable_log` is enabled or not. That's not good.

Reply:
Yes, should I write a dummy `#[instrument]`?

Reply:
I think so. We need a dummy macro, or write helper functions that only wraps the futures by `instrument` when the feature is on.